en/docs/references/concepts/compliance/ccpa.md

```diff
@@ -2,12 +2,14 @@
 
 The California Consumer Privacy Act (CCPA) is a bill passed by the California State Legislature, that came into effect on January 1, 2020. The act ensures that consumers are given more power and control over their personal data. Accordingly, CCPA compliance is a must for any organization that collects and processes personal information (PI) of California residents. According to the CCPA,  “Personal information means information that identifies, relates to, describes, is capable of being associated with, or could reasonably be linked, directly or indirectly, with a particular consumer or household.". This includes a broader spectrum of identifiers such as purchase events or download actions of a user as opposed to the Personally Identifiable Information (PII) which includes only aspects unique to the user such as the user's email address or social security number. 
 
-## Main clauses in CCPA and how WSO2 Identity Server complies to it
+---
+
+## Main clauses in CCPA and how WSO2 Identity Server complies with it
 
 1.  **The right to access Personal Information (PI)**
     This clause insists that California residents have the right to know the categories of personal information a business collects and sells to third parties. 
     The business should also be ready to disclose details regarding the **categories of third parties** with whom this personal information has been shared. 
-    Moreover, the consumer can request for specific information about them that has been collected and shared, and the business should ensure that the detail 
+    Moreover, the consumer can request specific information about them that has been collected and shared, and the business should ensure that the detail 
```

Comment:
I think it is better to have the "for" here. We use it when we are asking for something specific.